vmm/src/lib.rs

```diff
@@ -296,6 +296,32 @@ struct VmMigrationConfig {
     memory_manager_data: MemoryManagerSnapshotData,
 }
 
+macro_rules! add_device_to_vm {
```

Comment:
Honestly. I think I would rather go back to the version without the macros. I think the added complexity isn't worth it.

Reply:
Perfect, thanks for thequick feedback @rbradford!